### ISSUE TITLE
Flexible psyllid parameter setting

### ISSUE THREAD

**Issue by wcpettus** [project8/dragonfly#153]
When installing psyllid v1.6.0 without a corresponding dragonfly update, endpoint gets were returning errors (this is probably a good thing), but I think the code is too rigid in psyllid_provider.

For instance, the `get_trigger_configuration` method calls a series of named methods to get_fmt_<parameter_name>.  When we change these in psyllid, we have to recode the psyllid_provider instead of just manipulating a hardware config file.

A more flexible implementation would be a single-point `get_fmt_parameter(self, channel, parameter)` method that allows you to loop valid parameters in a list provided in the config file.  Maybe it needs to be a dict so you return int/float/string properly?  Maybe do something fancy to reformat # of packets into time.

Referring this to the DAQ WG @laroque.

**Comment by cclaessens** [comment]
I agree that that would be better. I would prefer to make that a new branch and pr and not include it in #154.
